src/java/org/apache/cassandra/service/consensus/TransactionalMode.java

```diff
@@ -112,7 +112,10 @@ public enum TransactionalMode
      * Execute writes through Accord skipping StorageProxy's normal write path. Ignores the provided consistency level
      * which makes Accord commit writes at ANY similar to Paxos with commit consistency level ANY.
      */
-    full(true, true, true, true, true);
+    full(true, true, true, true, true),
+
+    // For tests, Accord will read and be forced to do interop reads
+    test_interop_read(true, false, false, true, true);
```

Comment:
more states worry me... and even at community over code the feedback was to limit this enum...

Reply:
what is the reason for having test only states?  It's more testing that should never end up in prod, so what is the value we get?  Where do we test this really (it looks like we just take existing tests and run with these modes; so not really testing these modes)

Reply:
The `test_interop_read` code path forces all reads down the interop read path which otherwise would only be hit during migration. `unsafe_writes` is only used to test read repair in Accord because otherwise writes are forced through Accord in any "safe" mode.

`unsafe` doesn't really get used for anything and could go away entirely. It's not even really used for tests it's just what existed when there was only `TransactionStatement`.

Part of the problem is this configuration is an enum so you can't create a test configuration to get a specific.